src/main/java/hudson/plugins/throttleconcurrents/ThrottleQueueTaskDispatcher.java

```diff
@@ -61,83 +70,102 @@ public CauseOfBlockage canTake(Node node, Task task) {
     private CauseOfBlockage canTakeImpl(Node node, Task task) {
         final Jenkins jenkins = Jenkins.getActiveInstance();
         ThrottleJobProperty tjp = getThrottleJobProperty(task);
-        
+        List<String> pipelineCategories = categoriesForPipeline(task);
+
         // Handle multi-configuration filters
-        if (!shouldBeThrottled(task, tjp)) {
+        if (!shouldBeThrottled(task, tjp) && pipelineCategories.isEmpty()) {
```

Comment:
So re: this and later similar ugliness - I plan to rewrite `ThrottleQueueTaskDispatcher` pretty much completely before I'm done here, or at least rationalize it a lot better. This was me wedging things together until they worked, with clean up planned for afterwards.